### DISCUSSION TITLE
Rate of Change of the ROR?

### DISCUSSION THREAD

**Discussion by PKeyes505** [artisan-roaster-scope/artisan#705] [Q&A] [answered]
Is there an easy way to show the rate of change(Delta) of the RoR?

**Comment by CarefreeBuzzBuzz** [comment]
Please read the Quick Start Guide on setup and curves.  

Michael Herbert 
CarefreeBuzzBuzz

**Comment by poundy** [comment]
Just checking - do you really mean the RoR of the RoR itself? If so, that would more correctly be a rate of change, but no, there's no integral of the RoR and honestly I can't see its value

**Reply by PKeyes505** [reply]
I really do mean the RoR of the RoR.  I was thinking about Scott Rao recommending the declining but steady decrease in RoR and how to connect to a PID.  It seems the RoR of the RoR would be a fairly static (level) value and would require less adjustment.

**Reply by MAKOMO** [reply]
> It seems the RoR of the RoR would be a fairly static (level) value and would require less adjustment.

Well, in theory. Check the computed data to get an impression of reality. The amount of smoothing you need to apply even on excellent measuring systems will delay the signal that much that your PID will act much to late always.

**Comment by MAKOMO** [comment, marked as answer]
Here is a post that shows how to use symbolic formulas to compute the RoR of the RoR.

https://artisan-roasterscope.blogspot.com/2019/11/symbolic-formulas-basics-new-variables.html#identifyingFlicks

This one use it to identify flicks. You can add a virtual device along those lines and have its curve plotted during roasting, but its noise will render this completely useless.

**Reply by PKeyes505** [reply]
Thank you.  That seems like just the right post.
